packages/relay/src/lib/services/clientService/index.ts

```diff
@@ -0,0 +1,276 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import dotenv from 'dotenv';
+import findConfig from 'find-config';
+import { AccountId, Client, PrivateKey } from '@hashgraph/sdk';
+import { Logger } from 'pino';
+import { Registry, Counter } from 'prom-client';
+import { SDKClient } from '../../clients/sdkClient';
+
+export default class ClientService {
+  private transactionCount: number;
+  private errorCount: number;
+  private resetDuration: number;
+  private shouldReset: boolean;
+
+  private isEnabled: boolean;
```

Comment:
Q: What does this enable or disable? The whole SDK usage or just the batch reinitializations?

Reply:
Just batch reinitialisation. Maybe I should rename it to something like `isReinitEnabled` or something similar ?

Reply:
Renamed.